docs/src/apimanual.md

```diff
@@ -135,21 +135,38 @@ If `v` is a `VariableIndex` object, then `SingleVariable(v)` is simply the scala
 
 A more interesting function is [`ScalarAffineFunction`](@ref MathOptInterface.ScalarAffineFunction), defined as
 ```julia
-struct ScalarAffineFunction{T} <: AbstractFunction
-    variables::Vector{VariableIndex}
-    coefficients::Vector{T}
+struct ScalarAffineFunction{T} <: AbstractScalarFunction
+    terms::Vector{ScalarAffineTerm{T}}
     constant::T
 end
 ```
 
-If `x` is a vector of `VariableIndex` objects, then `ScalarAffineFunction([x[1],x[2]],[5.0,-2.3],1.0)` represents the function ``5x_1 - 2.3x_2 + 1``.
+The [`ScalarAffineTerm`](@ref MathOptInterface.ScalarAffineTerm) struct simply
+defines a pair of a variable and coefficient:
```

Comment:
Not sure which:
defines a pair of a variable index and a coefficient
defines a variable-coefficient pair